paket.template

```diff
@@ -0,0 +1,24 @@
+type file
```

Comment:
<s>Do you mind trying this locally? At present, I see exactly what I saw before even though I renamed `paket.template` to `ScriptCs.FSharp.fsproj.paket.template`.</s>

Turns out this whole time I've been looking in a folder I used at first and then abandoned but didn't clean. :(